### ISSUE TITLE
Visibly mark CRADLE studies in card list view

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#791]
# TL;DR
Studies which are CRADLE (TBD criteria, but roughly - data made immediately available to all Lookit researchers when parents give this permission) have a visual indication to families visiting the site

# Narrative
As a parent/guardian, I see and am excited by the possibility of contributing data to the whole research community
As a researcher, I benefit from deciding to share my data with other researchers by getting extra attention from families!

# Acceptance Criteria
[TBD]

- [ ] Tests for new feature
- [ ] QA on new feature
- [ ] Sign off from Kim

# Implementation Notes

Subject to the definition of CRADLE on Lookit

**Comment by mekline** [comment]
Decision made to handle this without new infrastructure - studies will include a 'corner graphic' identifying them as part of GARDEN
